src/NServiceBus.Core/Transports/TransportComponent.cs

```diff
@@ -67,6 +103,7 @@ public Configuration(SettingsHolder settings)
                 this.settings = settings;
 
                 settings.SetDefault(TransportConnectionString.Default);
+                settings.Set(new QueueBindings());
```

Comment:
should we add a note that we do this for backwards compatibility?

Reply:
Done